h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -1302,8 +1298,11 @@ public void learn() {
       for (int i = 0; i < bestModelsOfEachType.size(); i++)
         bestModelKeys[i] = bestModelsOfEachType.get(i)._key;
 
-      Job<StackedEnsembleModel> bestEnsembleJob = stack("StackedEnsemble_BestOfFamily", bestModelKeys);
+      Job<StackedEnsembleModel> bestEnsembleJob = stack("StackedEnsemble_BestOfFamily", bestModelKeys, true);
       pollAndUpdateProgress(Stage.ModelTraining, "StackedEnsemble build using top model from each algorithm type", seWork, this.job(), bestEnsembleJob, true);
+      
+      Job<StackedEnsembleModel> ensembleJob = stack("StackedEnsemble_AllModels", notEnsembles, false);
+      pollAndUpdateProgress(Stage.ModelTraining, "StackedEnsemble build using all AutoML models", seWork, this.job(), ensembleJob, true);
```

Comment:
@ledell is it a concern if I swap training order between BestOf and AllModels?
I do this to avoid having to recompute some of the predictions twice as `BestOfModels ⊂ AllModels`.
As SE are not time-bound, I don't see any inconvenient in doing this (just saying as AllModels is usually better than bestof so we definitely don't want to skip it).

Reply:
No, i don't have any issue with it and it sounds like a nice little optimization.